### PR TITLE
Remove Explore Super Menu AB Test code

### PR DESCRIPTION
**Pull request by chao-xian**
The test has concluded and all users now get the new header.

Trello https://trello.com/c/h1i6FA2i/540-remove-ab-test-code-from-government-frontend

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/controllers/content_items_controller.rb [hunk 1]
```diff
@@ -1,6 +1,5 @@
 class ContentItemsController < ApplicationController
   include Slimmer::Template
```
Comment (injms) [inline app/controllers/content_items_controller.rb:2]: If this is a revert of https://github.com/alphagov/government-frontend/pull/2156/, this line can be removed:

```suggestion
```

File: app/controllers/content_items_controller.rb [hunk 2]
```diff
@@ -13,10 +12,7 @@ class ContentItemsController < ApplicationController
   rescue_from PresenterBuilder::SpecialRouteReturned, with: :error_notfound
   rescue_from PresenterBuilder::GovernmentReturned, with: :error_notfound
 
-  before_action :set_explore_menu_response
-  after_action :set_slimmer_template
-
-  helper_method :explore_menu_variant, :explore_menu_variant_b?
+  slimmer_template "core_layout"
```
Comment (injms) [inline app/controllers/content_items_controller.rb:15]: The `core_layout` template is the default template in Slimmer - does it need to be set here?

File: app/controllers/content_items_controller.rb (hunk 1 above)
Comment (chao-xian) [reply app/controllers/content_items_controller.rb]: 👍

File: app/controllers/content_items_controller.rb (hunk 2 above)
Comment (chao-xian) [reply app/controllers/content_items_controller.rb]: 👍

**Review by injms — APPROVED** [review]
🎉

**Pull request merged**